src/platform/elements/picker/extras/base-picker-results/BasePickerResults.ts

```diff
@@ -84,7 +86,15 @@ export class BasePickerResults {
         }
         this.isLoading = false;
         this.ref.markForCheck();
-        setTimeout(() => this.overlay.updatePosition()); // @bkimball: This was added for Dylan Schulte, 9.18.2017 4:14PM EST, you're welcome!
+        setTimeout(() => {
+          this.overlay.updatePosition();
+          if (this.config.enableInfiniteScroll) {
+            let element: Element = this.getListElement();
+            if (element) {
+              element.addEventListener('scroll', this.scrollHandler);
```

Comment:
When they close the pickers, you will want to `element.removeEventListener(this.scrollHandler)`